frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -124,48 +130,94 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
   const ownedCRDs = spec.customresourcedefinitions.owned || [];
   const route = (name: string) => `/k8s/ns/${metadata.namespace}/${ClusterServiceVersionModel.plural}/${metadata.name}/${referenceForCRDDesc(_.find(ownedCRDs, {name}))}/new`;
 
-  return <div className="co-clusterserviceversion-details co-m-pane__body">
-    <div className="co-clusterserviceversion-details__section co-clusterserviceversion-details__section--info">
-      <div style={{marginBottom: '15px'}}>
-        { status.phase !== ClusterServiceVersionPhase.CSVPhaseSucceeded && <button disabled={true} className="btn btn-primary">Create New</button> }
-        { status.phase === ClusterServiceVersionPhase.CSVPhaseSucceeded && ownedCRDs.length > 1 && <Dropdown
-          buttonClassName="btn-primary"
-          title="Create New"
-          items={ownedCRDs.reduce((acc, crd) => ({...acc, [crd.name]: crd.displayName}), {})}
-          onChange={(name) => history.push(route(name))} /> }
-        { status.phase === ClusterServiceVersionPhase.CSVPhaseSucceeded && ownedCRDs.length === 1 && <Link to={route(ownedCRDs[0].name)} className="btn btn-primary">{`Create ${ownedCRDs[0].displayName}`}</Link> }
+  return <React.Fragment>
+    <ScrollToTopOnMount />
+
+    <div className="co-m-pane__body">
+      <SectionHeading text="ClusterServiceVersion Overview" />
+      <div className="co-m-pane__body-group">
```

Comment:
Split the sections using standard `co-m-pane__body` classes instead of custom ones.